pkg/cmd/server/api/v1/types_test.go

```diff
@@ -118,6 +118,8 @@ imageConfig:
   format: ""
   latest: false
 kind: MasterConfig
+kubeAdmissionControlPluginsOverride:
```

Comment:
if you're not starting a kubernetes master, the kubernetes plugin list doesn't apply, right? why wouldn't we move that setting under the kubernetesMasterConfig?